diff --git a/src/configwidget.cpp b/src/configwidget.cpp
index edf49c9..a7e2382 100644
--- a/src/configwidget.cpp
+++ b/src/configwidget.cpp
@@ -142,7 +142,7 @@ ConfigWidget::ConfigWidget()
         ADD_VALUE(vanishing_vars,Double,blue_team_vanishing,0,"Blue team")
         ADD_VALUE(vanishing_vars,Double,yellow_team_vanishing,0,"Yellow team")
         ADD_VALUE(vanishing_vars,Double,ball_vanishing,0,"Ball")
-    world=VarXML::read(world,(QDir::homePath() + QString("/.grsim.xml")).toStdString());
+    world=VarXML::read(world,(QDir::homePath() + QString("/robocup/tools/.grsim.xml")).toStdString());
 
 
     QDir dir;
@@ -201,7 +201,7 @@ ConfigWidget::ConfigWidget()
 }
 
 ConfigWidget::~ConfigWidget() {  
-   VarXML::write(world,(QDir::homePath() + QString("/.grsim.xml")).toStdString());
+   VarXML::write(world,(QDir::homePath() + QString("robocup/tools/.grsim.xml")).toStdString());
 }
 
 
